### ISSUE TITLE
MSFT-245[Registration] JAWS : Windows / Edge : Input field labels are announced twice

### ISSUE THREAD

**Issue by veshu** [veshu/testjiraimport#15]
**Steps to Reproduce**
1. Launch JAWS
2. Launch "https://accounts.topcoder.com/member/registration" in Windows / Edge
3. Navigate by virtual cursor [ARROW keys)
4. Navigate through input fields

**Expected Result**
Input fields' label should be announce only once

**Actual Result**
Note that each input field's label is announced twice continuously

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Edge
**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.3.1 Info and Relationships

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=1a9ec9e4%2D985a%2D449c%2D9e7a%2Dbb83cbe69ccb&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FArun%5FA11Y%2FJAWS%20Edge%20input%20field%20labels%20announced%20twice
